yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/share/ThirdPartyAttributeIssuingRequestBuilder.java

```diff
@@ -0,0 +1,43 @@
+package com.yoti.api.client.spi.remote.call.share;
+
+import com.yoti.api.client.AttributeDefinition;
+import com.yoti.api.client.spi.remote.IssuingAttribute;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ThirdPartyAttributeIssuingRequestBuilder {
+
+    private String issuanceToken;
+
+    private List<IssuingAttribute> issuingAttributes;
+
+    public ThirdPartyAttributeIssuingRequestBuilder() {
+        this.issuingAttributes = new ArrayList<>();
+    }
+
+    public ThirdPartyAttributeIssuingRequestBuilder withIssuanceToken(String issuanceToken) {
+        this.issuanceToken = issuanceToken;
+        return this;
+    }
+
+    public ThirdPartyAttributeIssuingRequestBuilder withIssuingAttribute(AttributeDefinition definition, String value) {
+        IssuingAttribute issuingAttribute = new IssuingAttribute(definition, value);
+        return withIssuingAttribute(issuingAttribute);
+    }
+
+    public ThirdPartyAttributeIssuingRequestBuilder withIssuingAttribute(IssuingAttribute issuingAttribute) {
+        this.issuingAttributes.add(issuingAttribute);
+        return this;
+    }
+
+    public ThirdPartyAttributeIssuingRequestBuilder withIssuingAttributes(List<IssuingAttribute> issuingAttributes) {
```

Comment:
Would be good if we annotated this to make sure users know that any previously added attributes will be overridden. Then if we do that, I think we should add annotations to the other methods too.

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/142/commits/9f85e1478a0c1ea43cbf1b24bd1bf5d3040a8691